### DISCUSSION TITLE
Mobile Issue with Terminal Theme

### DISCUSSION THREAD

**Discussion by Sunil-D-R** [GetPublii/Publii#1346] [Bug Report]
### Operating system

macOS Ventura Version 13.0

### Publii version

0.44.4 (build 16557)

### Issue type

Free theme

### Bug description

Having some issues with the mobile version though.
On some pages I either get no mobile menu and on others, I get the mobile menu but the heading text is squished up. 
<img width="350" alt="Screenshot 2024-02-07 at 3 36 19 PM" src="https://github.com/GetPublii/Publii/assets/142221930/a0b5e54c-6326-4ebc-bb42-cf642104d0ec">

### Steps to reproduce

Access news.promptengineering.org from mobile

**Comment by bobmitro** [comment]
URL please

**Reply by Sunil-D-R** [reply]
news.promptengineering.org

**Comment by bobmitro** [comment]
Unfortunately, I can't reproduce your issue; for me, the menu is displayed on all pages. I suspect that the long logo slogan might be causing your problem. So, you could try adding the following CSS rule via the Custom CSS tool:
```
.logo {
   line-height: 1;
   letter-spacing: -.05em;
}
```
